DNN Platform/DotNetNuke.Web.Client/Providers/DnnCompositeFileProcessingProvider.cs

```diff
@@ -34,6 +35,31 @@ public class DnnCompositeFileProcessingProvider : CompositeFileProcessingProvide
     {
         private readonly ClientResourceSettings clientResourceSettings = new ClientResourceSettings();
 
+        public override string MinifyFile(Stream fileStream, ClientDependencyType type)
+        {
+            Func<Stream, string> streamToString = stream =>
+            {
+                if (!stream.CanRead)
+                    throw new InvalidOperationException("Cannot read input stream");
+
+                if (stream.CanSeek)
+                    stream.Position = 0;
+
+                var reader = new StreamReader(stream);
+                return reader.ReadToEnd();
+            };
+
+            switch (type)

```

Comment:
I think `if` statements will be clearer here, getting rid of the ternary operator 

```
if (type == ClientDependencyType.Css && MinifyCss) 
{
    return CssHelper.MinifyCss(fileStream);
}

if (type == ClientDependencyType.Javascript && MinifyJs) 
{
    return JSMin.CompressJS(fileStream);
}

return streamToString(fileStream);
```

Reply:
In which case you won't need the `Func` either. Just inline the `streamToString`